sdcm/cluster.py

```diff
@@ -1792,6 +1798,23 @@ def node_setup(node):
         time_elapsed = time.time() - start_time
         self.log.debug('Setup duration -> %s s', int(time_elapsed))
 
+    def get_monitor_snapshot(self):
+        for n, node in enumerate(self.nodes):
+            try:
+                process.run('wget "https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-2.1.1-linux-x86_64.tar.bz2"',
+                            verbose=True)
+                process.run('tar xvfj phantomjs-2.1.1-linux-x86_64.tar.bz2',
+                            verbose=True)
+                process.run("cd phantomjs-2.1.1-linux-x86_64  && "
+                            "sed -e 's/}, 200);/}, 10000);/' examples/rasterize.js > rasterize.js && "
+                            "bin/phantomjs rasterize.js \"http://%s:3000/dashboard/db/scylla-per-server-metrics-1-6?from=%s&to=now\" \"%s/grafana-snapshot-%s.png\" && "
+                            "cd -" % (node.public_ip_address,
+                                      self.grafana_start_time,
+                                      self.logdir, n), shell=True, verbose=True)
+            except Exception, details:
+                self.log.error('Error taking monitor snapshot: %s',
+                               str(details))
```

Comment:
I tend to like splitting up commands in easier to understand chunks. So for what is worth, what you did is fine, but consider splitting up for added clarity.